traffic_portal/app/src/common/modules/table/profilesParamsCompare/table.profilesParamsCompare.tpl.html

```diff
@@ -0,0 +1,58 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div class="x_panel">
+    <div class="x_title">
+        <ol class="breadcrumb pull-left">
+            <li><a ng-click="navigateToPath('/profiles')">Profiles</a></li>
+            <li><a ng-click="navigateToPath('/profiles/' + profile1.id)">{{::profile1.name}}</a></li>
+            <li><a ng-click="navigateToPath('/profiles/' + profile2.id)">{{::profile2.name}}</a></li>
+            <li class="active">Compare Parameters</li>
+        </ol>
+        <div class="pull-right">
+            <button class="btn btn-default" title="Restore" ng-show="dirty" ng-click="restore()">Restore</button>
+            <button class="btn btn-primary" title="Update" ng-disabled="!dirty" ng-click="update()">Update</button>
```

Comment:
These buttons implicitly have `type="submit"`, which is fine for "Update", but the "Restore" button doesn't effect changes - it resets the view. It should have `type="reset"`.

Reply:
i really don't want any implicit meaning on either so i think best to just set to `type=button`